plugins/kontent-ai-plugin/scripts/setup.js

```diff
@@ -56,8 +74,10 @@ const importData = async (projectId, managementApiKey) => {
 
 console.log(`
   To use this starter, please create a new project in app.kontent.ai
-  and provide us with credentials need to set it up.
-  The required keys can be found in Project settings -> API KEYS and
+  and provide us with credentials need to set it up. If you want to use
+  already existing project, keep in mind that it should be clean to avoid
+  problems during importing data. This script provides you with an option to
+  clear it for you. The required keys can be found in Project settings -> API KEYS and
```

Comment:
```suggestion
  clear it for you. The required keys can be found in Project settings -> Api keys and
```

Just a small comment to align the casing of the menu option in the Kontent.ai UI